open_src/influx/httpd/config/config.go

```diff
@@ -76,6 +76,7 @@ type Config struct {
 	QueryMemoryLimitEnabled bool           `toml:"query-memory-limit-enabled"`
 	ChunkReaderParallel     int            `toml:"chunk-reader-parallel"`
 	ReadBlockSize           toml.Size      `toml:"read-block-size"`
+	DisableNoTimeFilter     bool           `toml:"disable-query-without-time-filter"`
```

Comment:
add the definition `disable-query-without-time-filter` to the config file

Reply:
> add the definition `disable-query-without-time-filter` to the config file
sorry, i will add it immediately